config.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+from typing import List
+
+from pydantic import BaseModel, Field
+
+
+log = logging.getLogger(__name__)
+
+
+class Config(BaseModel):
+    input_size: int = 1
+    input_variable: List[str] = Field(default_factory=lambda: ['runoff_mm'])
```

Comment:
Are these input and output variables backwards?

Reply:
I think I committed an old Config.py instance by mistake. I removed the input and output vars from this file to reduce complexity